client/src/components/ZestimateAdditionalInfo.jsx

```diff
@@ -1,55 +1,80 @@
 import React from 'react';
 import ComparableHomes from './ComparableHomes.jsx';
 import MarketAppreciation from './MarketAppreciation.jsx';
+import ZestDeepDiveButton from './ZestDeepDiveButton.jsx';
+import styled from 'styled-components';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle, faHome, faAngleDown} from '@fortawesome/free-solid-svg-icons';
+
+library.add(faHome, faCircle, faAngleDown);
+
+const AdditionalZestimateInformation = styled.div`
+display: flex;
+`;
+const ZestDeepDive = styled.div``;
+
+const ZestDeepDiveIntro = styled.div`
+  text-align: left;
```

Comment:
Inconsistent tabbing with styled-components. Should either have all of them tabbed once, or not tabbed at all.

Reply:
@leightonchen Good eye